_harp/css/components/sections/_main-preview.scss

```diff
@@ -21,6 +22,7 @@
 #preview-player {
   margin: 0 auto 30px;
   width: 100%;
+  padding-top: $preview-video-ratio * 100%;
```

Comment:
That works for me for this. In the grand scheme of video.js though that feels a little hacky, so we might look out for a way to better support this use case for video.js users (built-in if somehow possible). It feels like this usage of the player is growing more common.